docs/start/install.rst

```diff
@@ -261,10 +536,20 @@ know how to resolve them. If none of these help you, please ask a question on
 
 .. _`PyPI`: https://pypi.org/
 
-.. _`AppVeyor`: https://ci.appveyor.com/project/h2oops/datatable/history
+.. _`AppVeyor`: https://ci.appveyor.com/project/h2oops/datatable/
```

Comment:
All the builds are available under the "History" tab, so I wouldn't change this link.